### PR TITLE
promote ci-kubernetes-e2e-ubuntu-gce-containerd to release blocking

### PR DESCRIPTION
**Pull request by dims**
this job has served us well, please promote it to release blocking.

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml [hunk 1]
```diff
@@ -405,7 +405,7 @@ periodics:
           - --timeout=50m
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200602-c1964ea-master
   annotations:
-    testgrid-dashboards: sig-release-master-informing, google-gce, google-gci
+    testgrid-dashboards: sig-release-master-blocking, google-gce, google-gci
     testgrid-tab-name: gce-ubuntu-master-containerd
     testgrid-num-failures-to-alert: '6'
     testgrid-alert-email: kubernetes-release-team@googlegroups.com
```
Comment (justaugustus) [inline config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml:411]: Which SIG is the primary responsible party for this job? They need to be alerted on failures as well.
/hold

Comment (dims) [reply config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml]: let's use the sig-testing alerts

**Pull request merged**